src/main/resources/static/js/keys.js

```diff
@@ -1,2 +1,2 @@
  const googleMapAPI = "AIzaSyCMpJ-NawmnjMOSlkw5uXkLYjDTSzEmvag"
- const fileStackAPI = "AVQDSwWMqSSOe8rY5RPkNz"
```

Comment:
Please keep the AVQD key for testing. That will avoid burning through the one assigned to the live site.